### PR TITLE
Updated make files 

### PR DESCRIPTION
**Pull request by lawrenceccheung**
I updated the `Make.package` and the `make.incflo` files in case anybody is unable to use cmake for compilation.

Lawrence

### CODE REVIEW

**Comment by sayerhs** [comment]
@lawrenceccheung we are looking to deprecate the `Make` file pathway. Can you explain the issue with CMake builds for you?

File: src/utilities/PlaneAveraging.cpp [hunk 1]
```diff
@@ -63,7 +63,7 @@ void PlaneAveraging::plot_line_average_text(std::string filename, int step, Real
     }
 
     for(int i=0;i<ncell_line;++i){
-        outfile << step << ", " << std::scientific << time << ", " << line_xcentroid[i];
+        outfile << step << ", " << std::scientific << std::setprecision(10) << time << ", " << line_xcentroid[i];
```
Comment (sayerhs) [inline src/utilities/PlaneAveraging.cpp]: unrelated change to the pullrequest

**Comment by lawrenceccheung** [comment]
There's a requirement in the `CMakeLists.txt` for cmake >= 3.14 (also is in the amrex submodule).  The currently installed cmake at Sandia goes up to 3.12, so I'm currently using make while I ask the HPC people to upgrade cmake.

File: src/utilities/PlaneAveraging.cpp (hunk 1 above)
Comment (lawrenceccheung) [reply src/utilities/PlaneAveraging.cpp]: I undid the change to `PlaneAveraging`, will split it to a different pull request.
